src/States/atDanceGameState.js

```diff
@@ -41,12 +64,29 @@ function AtDanceGame() {
 
 		//draws table
 		player = game.add.sprite(createPlayer.x, createPlayer.y, 'playerImg');
+		game.physics.arcade.enable(player);
+
+
+		item1 = game.add.sprite(100, 100, 'itemImg1');
+		game.physics.arcade.enable(item1);
+		item2 = game.add.sprite(200, 200, 'itemImg2');
+		game.physics.arcade.enable(item2);
+		item3 = game.add.sprite(300, 300, 'itemImg3');
+		game.physics.arcade.enable(item3);
 
 	}
 
 	function update() {
```

Comment:
:space_invader: No need for the new line spacing at the start and end of `update()`